e2e-tests/src/test/high_latency.rs

```diff
@@ -0,0 +1,86 @@
+use std::cmp::max;
+
+use aleph_client::{
+    pallets::session::SessionApi,
+    waiting::{AlephWaiting, BlockStatus},
+};
+use log::info;
+
+use crate::{config::setup_test, synthetic_network::set_out_latency};
+
+#[tokio::test]
+pub async fn high_out_latency() -> anyhow::Result<()> {
+    let config = setup_test();
+
+    let connections = config.create_signed_connections().await;
+    info!("waiting for at least session 3");
+    for connection in &connections {
+        if connection.get_session(None).await < 3 {
```

Comment:
Why is this check necessary? (Probably relates to my question about `max_session`)

Reply:
it's not an issue - I can remove it, I needed it to speed local tests.